src/oncoexporter/cda/cda_table_importer.py

```diff
@@ -30,7 +31,9 @@ class CdaTableImporter(CdaImporter):
     :type use_cache: bool
     """
 
-    def __init__(self, cohort_name: str, query: str = None, query_obj: Q = None, use_cache=False):
+    def __init__(self, cohort_name: str, query: str = None, query_obj: Q = None,
+                 use_cache=False,
+                 icdo_to_ncit_map_url='https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/Mappings/ICD-O-3_Mappings/ICD-O-3.1-NCIt_Morphology_Mapping.txt'):
```

Comment:
Generally I would like to encourage to not hide our treatments of mappings so deep in code (esp. not hardcode URLs), and ETL them through ["Monarch mapping commons"](https://github.com/monarch-initiative/monarch-mapping-commons) and then use the SSSOM files instead. Having a simple API that reading a SSSOM mapping and then perform mapping operations seems to be the next logical step for us!

Reply:
Yes, exactly, with the following motivation:

1. Other projects that need the mapping do not have to write ETL / parsing code and we have a centralised placed for all our mapping needs across the organisation
2. We can make our assumption about the mapping explicit during ETL (e.g. interpreting the subject and object match as an "exact" match)

But independent of that, I would avoid hard coding URLs in the code, and use the data pipeline to obtain the mapping set instead (e.g. nextflow, makefile, snakemake etc).

Reply:
This is useful feedback, and I agree that folding this mapping into some mapping commons is the technically superior solution. 

For the sake of conversation though, it's worth considering why these patterns (hard coding an overridable default URL into ETL code) exist. 

Consider a common scenario in ETL: this remote resource (here a mapping file) has an updated version that the oncoexporter user would like to use. 

In the hacky hard-coded default URL solution, all that is required is for the oncoexporter user to call the existing code with a different argument:

```
CdaDiseaseFactory(url="https://some/new/mapping/version.txt")
```

In the mapping commons solution, the oncoexporter user must examine the URL, surmise that the resource is coming from the mapping commons, go find the mapping commons, study the code, make a PR, wait for the PR to be merged and for the new mapping to appear. That is a lot of friction, which is quite expensive. Again, I agree that the mapping solution is the technically superior solution, just pointing out that there are often invisible costs to these such solutions...

Reply:
This is good feedback! I will take it under consideration; my immediate response is that the "I would like to use another version" desire should arguably be had at organisation- rather than project level, to avoid different projects across the same org applying different mappings (resulting in different displays and results across apps). So from the individual tool, the desire should not even come up. But especially for research projects, this view is probably impractical. I cede my point here :) Thanks for taking the time for your thoughtful response!